documentation/staging/content/userguide/managing-domains/accessing-the-domain/port-forward.md

```diff
@@ -63,19 +63,52 @@ This behavior depends on your version and domain resource configuration:
   [schema](https://github.com/oracle/weblogic-kubernetes-operator/blob/main/documentation/domains/Domain.md).
 
 * For Istio-enabled domains running Istio versions prior to 1.10,
-  the operator already adds a
-  network channel with a `localhost` listen address for each
-  existing port. This means that no additional configuration is required
-  to enable port forwarding when Istio is enabled.
+  you must add an additional network channel to the WebLogic Administration Server 
+  configured with the following attributes:
+  * Protocol defined as `t3`.
+  * Listen address defined with `localhost`.
+  * Listen port. Note, you should choose a port value that does not conflict with any ports defined 
+  in any of the additional network channels created for use with Istio versions prior to v1.10.
   For more details, see [Added network channels for Istio versions prior to v1.10]({{< relref "/userguide/istio/istio#added-network-channels-for-istio-versions-prior-to-v110" >}}).
+  * Enable `HTTP` protocol for this network channel.
+  * Do _NOT_ set an `external listen address` or `external listen port`.
+  
+For example, here is a snippet of a WebLogic domain `config.xml` file for channel `PortForward` for the Admin Server
+```xml
+<server>
+  <name>admin-server</name>
+  <network-access-point>
+    <name>PortForward</name>
+    <protocol>t3</protocol>
+    <listen-address>localhost</listen-address>
+    <listen-port>7890</listen-port>
+    <http-enabled-for-this-protocol>true</http-enabled-for-this-protocol>
+  </network-access-point>
+</server>
+```
+For Model in Image (MII) and Domain Home in Image (DII), here is a snippet model configuration for channel `PortForward` for the Admin Server
```

Comment:
Domain Home in Image -> Domain in Image
Period at the end of the sentence. -> ...for the Administration Server.